app/javascript/gobierto_dashboards/webapp/components/SearchFilter.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="gobierto_dashboards-search-container">
+    <label class="gobierto_dashboards-search-container-label">
+      {{ labelSearch }}:
+    </label>
+    <div class="gobierto_dashboards-search-container-wrapper-input">
+      <div class="search-box">
+        <i class="fas fa-search gobierto_dashboards-search-btn-search" />
+        <input
+          v-model="search"
+          type="text"
+          :placeholder="labelPlaceholder"
+          class="gobierto_dashboards-search-container-input"
+          @input="handlerFilterItems"
+        >
+        <i
+          class="fas fa-times gobierto_dashboards-search-btn-clear"
+          @click="handlerClearSearch"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { EventBus } from "../mixins/event_bus";
+export default {
+  name: 'SearchFilter',
+  props: {
+    data: {
+      type: Array,
+      default: () => []
+    },
+    searchType: {
+      type: String,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      search: '',
+      labelSearch: I18n.t('gobierto_dashboards.dashboards.contracts.search'),
+      labelPlaceholder: I18n.t('gobierto_dashboards.dashboards.contracts.search_placeholder')
+    }
+  },
+  methods: {
+    handlerFilterItems(event) {
```

Comment:
I see the logic of searching is duplicated between this method and `refreshSummaryData`, could you think in a way to share this behaviour? Ping me if you need to discuss (I thought I sent this comment yesterday but I can't find it)